assembl/static2/css/components/posts.scss

```diff
@@ -340,19 +351,25 @@ $deleted-post-border: 1px solid $light-grey !default;
   }
 
   .answer-form {
-    background-color: $first-color-light;
+    width: 90%;
```

Comment:
couldn't it be 100% width with external margin instead ?

Reply:
I tried, it's not working